### PR TITLE
refactor(rebalancing): refactor rebalancing doc

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

**Comment by wivwiv** [comment]
Defaults for evacuation:
```bash
    #{
        server_reference => undefined, %% do not set this prop at all
        conn_evict_rate => ?DEFAULT_CONN_EVICT_RATE, %% 500 conn/sec
        sess_evict_rate => ?DEFAULT_SESS_EVICT_RATE, %% 500 sess/sec
        wait_takeover => ?DEFAULT_WAIT_TAKEOVER, %% 60 sec
        migrate_to => undefined %% migrate sessions to all other available nodes
    }.
```
For rebalancing:
```bash
    #{
        conn_evict_rate => ?DEFAULT_CONN_EVICT_RATE, %% 500 conn/sec
        abs_conn_threshold => ?DEFAULT_ABS_CONN_THRESHOLD, %%  1000
        rel_conn_threshold => ?DEFAULT_REL_CONN_THRESHOLD, %% 1.1

        sess_evict_rate => ?DEFAULT_SESS_EVICT_RATE, %% 500 sess/sec
        abs_sess_threshold => ?DEFAULT_ABS_SESS_THRESHOLD, %% 1000
        rel_sess_threshold => ?DEFAULT_REL_SESS_THRESHOLD, %% 1.1

        wait_health_check => ?DEFAULT_WAIT_HEALTH_CHECK, %% 60 sec
        wait_takeover => ?DEFAULT_WAIT_TAKEOVER, %% 60 sec

        nodes => all_nodes() %% try to rebalance all nodes, i.e. the whole cluster
    }.
```

File: zh_CN/advanced/rebalancing.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
-# 集群负载重平衡与节点疏散
+# 节点疏散与集群负载重平衡
 
-集群负载重平衡是将客户端与会话从一组节点强行迁移到其他节点的行为，而节点疏散则是要将所有连接与会话从某个节点迁离的行为。
+MQTT 是有状态的长连接接入协议，因此连接建立后不会轻易断开。但在某些情况下，比如需要对集群节点进行升级、维护或扩容，此时就需要断开已有连接再进行重连，为便于用户的运维操作，EMQX 提供了节点疏散和集群负载重平衡功能。
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
MQTT 是有状态的长连接接入协议，连接建立后不会轻易断开，因此集群节点的升级、维护和扩容将变得更有挑战性。为便于用户的集群运维操作，EMQX 提供了节点疏散和集群负载重平衡功能。
```

File: zh_CN/advanced/rebalancing.md [hunk 2]
```diff
@@ -34,30 +38,42 @@ emqx_ctl rebalance start --evacuation \
...
 |---------------------|------------------|-------------|
-| `--redirect-to` | string | 具体可参考[MQTT 5.0 协议 - 服务器重定向(Server redirection)](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255)章节。MQTT 5.0 协议的客户端使用该属性重连 |
-| `--conn-evict-rate` | positive integer | 客户端每秒断开连接速度。 |
-| `--migrate-to` | string | 目标节点列表，多个节点以逗号或空格分隔。 |
-| `--wait-takeover` | positive integer | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
-| `--sess-evict-rate` | positive integer | `wait-takeover` 之后每秒会话疏散速度。 |
+| `--redirect-to` | 字符 | 重连时的重定向服务器地址，针对 MQTT 5.0 客户端；具体可参考 [MQTT 5.0 协议 - 服务器重定向(Server redirection) ](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255) |
+| `--conn-evict-rate` | 正整数 | 客户端每秒断开连接速度。 |
+| `--migrate-to` | 字符 | 目标节点列表，多个节点以逗号或空格分隔。 |
+| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间；单位： 秒，如不配置，则 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间；单位： 秒，默认为 60 秒 |
```

File: zh_CN/advanced/rebalancing.md [hunk 3]
```diff
@@ -34,30 +38,42 @@ emqx_ctl rebalance start --evacuation \
...
+| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后每秒会话疏散速度。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后每秒会话疏散速度。 默认为 500 会话/秒 |
```

File: zh_CN/advanced/rebalancing.md [hunk 4]
```diff
@@ -34,30 +38,42 @@ emqx_ctl rebalance start --evacuation \
...
-| 配置项          | 类型           | 描述 |
+| 配置项       | 类型           | 描述 |
 |---------------------|------------------|-------------|
-| `--redirect-to` | string | 具体可参考[MQTT 5.0 协议 - 服务器重定向(Server redirection)](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255)章节。MQTT 5.0 协议的客户端使用该属性重连 |
-| `--conn-evict-rate` | positive integer | 客户端每秒断开连接速度。 |
-| `--migrate-to` | string | 目标节点列表，多个节点以逗号或空格分隔。 |
-| `--wait-takeover` | positive integer | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
-| `--sess-evict-rate` | positive integer | `wait-takeover` 之后每秒会话疏散速度。 |
+| `--redirect-to` | 字符 | 重连时的重定向服务器地址，针对 MQTT 5.0 客户端；具体可参考 [MQTT 5.0 协议 - 服务器重定向(Server redirection) ](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255) |
+| `--conn-evict-rate` | 正整数 | 客户端每秒断开连接速度。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--conn-evict-rate` | 正整数 | 客户端每秒断开连接速度。默认为 500 连接/秒 |
```

File: zh_CN/advanced/rebalancing.md [hunk 5]
```diff
@@ -34,30 +38,42 @@ emqx_ctl rebalance start --evacuation \
...
+| `--migrate-to` | 字符 | 目标节点列表，多个节点以逗号或空格分隔。 |
+| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间；单位： 秒，如不配置，则 |
+| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后每秒会话疏散速度。 |
 
-示例：
+<!-- TODO 参数默认值-->
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
```

File: zh_CN/advanced/rebalancing.md [hunk 6]
```diff
@@ -34,30 +38,42 @@ emqx_ctl rebalance start --evacuation \
...
+./bin/emqx_ctl rebalance start --evacuation \
+	--wait-takeover 200 \
+	--conn-evict-rate 30 \
+	--sess-evict-rate 30 \
+	--migrate-to "emqx2@127.0.0.1 emqx3@127.0.0.1"
 Rebalance(evacuation) started
 ```
 
-获取疏散状态的命令如下：
+该命令会以每秒 `30` 个连接的速度断开现有客户端，所有连接断开后将等待 `200` 秒，之后以每秒 `30` 个会话的速度迁移现有会话到 `emqx2@127.0.0.1` 与 `emqx3@127.0.0.1`。
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
该命令会以每秒 `30` 个连接的速度断开现有客户端，所有连接断开后将等待 `200` 秒，期间会将客户端会话迁移到重连节点，之后以每秒 `30` 个会话的速度迁移剩余会话到 `emqx2@127.0.0.1` 与 `emqx3@127.0.0.1` 节点。
```

File: zh_CN/advanced/rebalancing.md [hunk 7]
```diff
@@ -74,37 +90,92 @@ Channel statistics:
...
+{"data":[],"code":0}
 ```
+
+请求体中可包含以下字段：
+
+| 字段              | 类型   | 是否必填项 | 说明                                                         |
+| ----------------- | ------ | ---------- | ------------------------------------------------------------ |
+| `nodes`           | 字符   | 是         | 节点名称                                                     |
+| `redirect_to`     | 正整数 | 是         | 重连时的重定向服务器地址，仅适用 MQTT 5.0 客户端<br>具体可参考 [MQTT 5.0 协议 - 服务器重定向(Server redirection) ](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255) |
+| `conn_evict_rate` | 正整数 | 是         | 客户端每秒断开连接速度。                                     |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `conn_evict_rate` | 正整数 | 是         | 客户端每秒断开连接速度。默认为 500 连接/秒。                                     |
```

File: zh_CN/advanced/rebalancing.md [hunk 8]
```diff
@@ -74,37 +90,92 @@ Channel statistics:
...
+| `migrate_to`      | 正整数 | 是         | 目标节点列表，多个节点以逗号或空格分隔。                     |
+| `wait_takeover`   | 正整数 | 否         | 所有连接断开后，等待客户端重连以接管会话的时间；单位： 秒    |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `wait_takeover`   | 正整数 | 否         | 所有连接断开后，等待客户端重连以接管会话的时间；单位： 秒 ，默认为 60 秒。   |
```

File: zh_CN/advanced/rebalancing.md [hunk 9]
```diff
@@ -74,37 +90,92 @@ Channel statistics:
...
+| `sess_evict_rate` | 正整数 | 否         | `wait-takeover` 之后每秒会话疏散速度。                       |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `sess_evict_rate` | 正整数 | 否         | `wait-takeover` 之后每秒会话疏散速度。默认为 500 会话/秒。                       |
```

File: zh_CN/advanced/rebalancing.md [hunk 10]
```diff
@@ -74,37 +90,92 @@ Channel statistics:
...
+
+```bash
+curl -v -u admin:public -H "Content-Type: application/json" -X POST 'http://127.0.0.1:8081/api/v4/load_rebalance/emqx1@127.0.0.1/evacuation/stop'
+
+{"data":[],"code":0}
+```
+
+## 集群负载重平衡
+
+同样由于 MQTT 是有状态的长连接接入协议，已连接节点会保持连接状态。即使在节点扩容场景下，现有节点的连接也不会自动切换到新增节点中，因此如果没有大量的新客户端接入，新增节点会长时间处于负载不足的状态。此时，您需要手动将连接从高负载节点迁移到低负载节点，实现集群负载均衡。
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]: 已连接客户端会保持连接状态 = 长连接。
```suggestion
同样由于 MQTT 是有状态的长连接接入协议，即使在节点扩容后，现有节点的连接也不会自动切换到新增节点中，因此如果没有大量的新客户端接入，新增节点会长时间处于负载不足的状态。此时，您需要手动将连接从高负载节点迁移到低负载节点，实现集群负载均衡。
```

File: zh_CN/advanced/rebalancing.md [hunk 11]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
-| `--wait-health-check` | positive integer | 等待 LB 将源节点从活跃的后端节点列表中移除的时间（单位为 秒），超过指定等待时间后，重平衡任务将启动。 |
-| `--conn-evict-rate` | positive integer | 源节点客户端每秒断开连接速度。 |
-| `--abs-conn-threshold` | positive integer | 用于检查连接平衡的绝对阈值。                                 |
-| `--rel-conn-threshold` | number > 1.0     | 用于检查连接平衡的相对阈值。                                 |
-| `--wait-takeover` | positive integer | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
- | `--sess-evict-rate` | positive integer | `wait-takeover` 之后源节点每秒会话疏散速度。 |
-| `--abs-sess-threshold` | positive integer | 用于检查会话连接平衡的绝对阈值。                             |
-| `--rel-sess-threshold` | number > 1.0     | 用于检查会话连接平衡的相对阈值。                             |
+| `--nodes` | 字符 | 参与负载重平衡的节点列表，以空格或逗号分隔，调度节点（即运行命令的节点）可以不在列表中。 |
+| `--wait-health-check` | 正整数 | 等待 LB 将源节点从活跃的后端节点列表中移除的时间（单位为 秒），超过指定等待时间后，重平衡任务将启动。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--wait-health-check` | 正整数 | 等待 LB 将源节点从活跃的后端节点列表中移除的时间（单位为 秒，默认 60 秒），超过指定等待时间后，重平衡任务将启动。 |
```

File: zh_CN/advanced/rebalancing.md [hunk 12]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
+| `--conn-evict-rate` | 正整数 | 源节点客户端每秒断开连接速度。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--conn-evict-rate` | 正整数 | 源节点客户端每秒断开连接速度。 默认 500 连接/秒。|
```

File: zh_CN/advanced/rebalancing.md [hunk 13]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
+| `--abs-conn-threshold` | 正整数 | 用于检查连接平衡的绝对阈值。                                 |
+| `--rel-conn-threshold` | 数字<br/>> 1.0 | 用于检查连接平衡的相对阈值。                                 |
+| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒，默认 60 秒）。 |
```

File: zh_CN/advanced/rebalancing.md [hunk 14]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
+| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后源节点每秒会话疏散速度。 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后源节点每秒会话疏散速度。默认 500 会话/秒。 |
```

File: zh_CN/advanced/rebalancing.md [hunk 15]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
-| `--rel-conn-threshold` | number > 1.0     | 用于检查连接平衡的相对阈值。                                 |
-| `--wait-takeover` | positive integer | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
- | `--sess-evict-rate` | positive integer | `wait-takeover` 之后源节点每秒会话疏散速度。 |
-| `--abs-sess-threshold` | positive integer | 用于检查会话连接平衡的绝对阈值。                             |
-| `--rel-sess-threshold` | number > 1.0     | 用于检查会话连接平衡的相对阈值。                             |
+| `--nodes` | 字符 | 参与负载重平衡的节点列表，以空格或逗号分隔，调度节点（即运行命令的节点）可以不在列表中。 |
+| `--wait-health-check` | 正整数 | 等待 LB 将源节点从活跃的后端节点列表中移除的时间（单位为 秒），超过指定等待时间后，重平衡任务将启动。 |
+| `--conn-evict-rate` | 正整数 | 源节点客户端每秒断开连接速度。 |
+| `--abs-conn-threshold` | 正整数 | 用于检查连接平衡的绝对阈值。                                 |
+| `--rel-conn-threshold` | 数字<br/>> 1.0 | 用于检查连接平衡的相对阈值。                                 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--rel-conn-threshold` | 数字<br/>> 1.0 | 用于检查连接平衡的相对阈值。默认为 1.1                                 |
```

File: zh_CN/advanced/rebalancing.md [hunk 16]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
-| `--abs-conn-threshold` | positive integer | 用于检查连接平衡的绝对阈值。                                 |
-| `--rel-conn-threshold` | number > 1.0     | 用于检查连接平衡的相对阈值。                                 |
-| `--wait-takeover` | positive integer | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
- | `--sess-evict-rate` | positive integer | `wait-takeover` 之后源节点每秒会话疏散速度。 |
-| `--abs-sess-threshold` | positive integer | 用于检查会话连接平衡的绝对阈值。                             |
-| `--rel-sess-threshold` | number > 1.0     | 用于检查会话连接平衡的相对阈值。                             |
+| `--nodes` | 字符 | 参与负载重平衡的节点列表，以空格或逗号分隔，调度节点（即运行命令的节点）可以不在列表中。 |
+| `--wait-health-check` | 正整数 | 等待 LB 将源节点从活跃的后端节点列表中移除的时间（单位为 秒），超过指定等待时间后，重平衡任务将启动。 |
+| `--conn-evict-rate` | 正整数 | 源节点客户端每秒断开连接速度。 |
+| `--abs-conn-threshold` | 正整数 | 用于检查连接平衡的绝对阈值。                                 |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--abs-conn-threshold` | 正整数 | 用于检查连接平衡的绝对阈值。默认为 1000。                                 |
```

File: zh_CN/advanced/rebalancing.md [hunk 17]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
+| `--rel-conn-threshold` | 数字<br/>> 1.0 | 用于检查连接平衡的相对阈值。                                 |
+| `--wait-takeover` | 正整数 | 所有连接断开后，等待客户端重连以接管会话的时间（单位为 秒）。 |
+| `--sess-evict-rate` | 正整数 | `wait-takeover` 之后源节点每秒会话疏散速度。 |
+| `--abs-sess-threshold` | 正整数 | 用于检查会话连接平衡的绝对阈值。                             |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--abs-sess-threshold` | 正整数 | 用于检查会话连接平衡的绝对阈值。默认为 1000。                             |
```

File: zh_CN/advanced/rebalancing.md [hunk 18]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
+| `--rel-sess-threshold` | 数字<br/>> 1.0 | 用于检查会话连接平衡的相对阈值。                             |
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
| `--rel-sess-threshold` | 数字<br/>> 1.0 | 用于检查会话连接平衡的相对阈值。默认为 1.1                             |
```

File: zh_CN/advanced/rebalancing.md [hunk 19]
```diff
@@ -120,30 +191,43 @@ rebalance start \
...
 ```
 
 类似的规则也适用于会话的连接断开。
 
-示例：
+#### 示例
 
-```
-./bin/emqx_ctl rebalance start --wait-health-check 10 --wait-takeover 60  --conn-evict-rate 5 --sess-evict-rate 5 --abs-conn-threshold 30 --abs-sess-threshold 30 --nodes "emqx1@127.0.0.1 emqx2@127.0.0.1 emqx3@127.0.0.1"
+如希望在 `emqx@127.0.0.1`、`emqx2@127.0.0.1` 与 `emqx3@127.0.0.1` 三个节点之间实现负载重平衡，可通过如下代码实现：
```
Comment (wivwiv) [inline zh_CN/advanced/rebalancing.md]:
```suggestion
如希望在 `emqx@127.0.0.1`、`emqx2@127.0.0.1` 与 `emqx3@127.0.0.1` 三个节点之间实现负载重平衡，可使用如下命令操作：
```

**Review by LenaLenaPan — DISMISSED** [review]

**Comment by LenaLenaPan** [comment]
@Meggielqk please also help to sync the English files as per the Chinese version here.

**Review by LenaLenaPan — APPROVED** [review]

**Pull request merged**